glean-core/src/glean.udl

```diff
@@ -195,3 +195,11 @@ interface TimespanMetric {
 
     i32 test_get_num_recorded_errors(ErrorType error, optional string? ping_name = null);
 };
+
+interface BooleanMetric {
+    constructor(CommonMetricData meta);
+
+    void set(boolean value);
+
+    boolean? test_get_value(optional string? ping_name = null);
```

Comment:
Does this need a function to return the number of recorded errors also?

Reply:
There are no errors for boolean metrics!
(well, technically there are for labeled booleans and because of how this is build it's required later, but that comes with the labeled ones)